pkg/runner/step_context.go

```diff
@@ -557,7 +557,12 @@ func (sc *StepContext) execAsComposite(ctx context.Context, step *model.Step, _
 			}
 		}
 
-		stepClone.Run = strings.ReplaceAll(stepClone.Run, "${{ github.action_path }}", filepath.Join(containerActionDir, actionName))
+		if stepClone.Env == nil {
+			stepClone.Env = make(map[string]string)
+		}
+		actionPath := filepath.Join(containerActionDir, actionName)
+		stepClone.Env["GITHUB_ACTION_PATH"] = actionPath
+		stepClone.Run = strings.ReplaceAll(stepClone.Run, "${{ github.action_path }}", actionPath)
```

Comment:
I claim that this is call to `strings.ReplaceAll` is a bug. But fixing it is out of scope for this PR, and I don't understand enough of how the processing works to understand how to fix it.

Ideally setting `github.action_path` (the object thingy that otto gets) should be enough such that otto is able to evaluate the thing and replace it properly.

If someone writes any of these, it won't work:
```
${{  github.action_path }}
${{ github.action_path}}
${{github.action_path}}
${{ github["action_path"] }}
${{ github['action_path'] }}
```

And I bet that all of these would work in the GitHub Action runners environment.

Reply:
If it's not in scope of this PR I would just leave it for now, since I also don't remember whole code to tell exactly why it's needed or what does it do.

Reply:
Oh, i know what it does since my original approach broke the test that I wrote. I just don't know how to get it to not need to be there ☹️.

I'm pointing it out so that it's not entirely forgotten. I suppose at some point I'll probably file a bug for it.